packages/docs-site/src/components/presenters/post-article/post-article.scss

```diff
@@ -0,0 +1,3 @@
+.post-article {
+  width: 0;
```

Comment:
This is a temporary layout quick fix. The `post-article` component needs styling from the ground up.